### PR TITLE
Adding DTT_ Constants 

### PR DESCRIPTION
**Pull request by chenss3**
Fixes: #902

Added in the flag constants DTT_TEXTCOLOR, DTT_BORDERCOLOR, DTT_SHADOWCOLOR, DTT_SHADOWTYPE, DTT_SHADOWOFFSET, DTT_BORDERSIZE, DTT_FONTPROP, DTT_COLORPROP, DTT_STATEID, DTT_CALCRECT, DTT_APPLYOVERLAY, DTT_GLOWSIZE, DTT_CALLBACK, DTT_COMPOSITED  into a new DTTOPTS flags enum. 

Metadata diff:
```
Windows.Win32.UI.Controls.DTTOPTS_FLAGS added
Windows.Win32.UI.Controls.DTTOPTS.dwFlags...System.UInt32 => Windows.Win32.UI.Controls.DTTOPTS_FLAGS
```

### CODE REVIEW

**Comment by elachlan** [comment]
@chenss3 can you please provide the diff from the build in the issue summary?

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -30318,5 +30318,76 @@
...
+      {
+        "name": "DTT_CALLBACK",
+        "value": "0x1000"
+      },
+      {
+        "name": "DTT_COMPOSITED",
+        "value": "0x2000"
+      },
+      {
+        "name": "DTT_VALIDBITS"
```
Comment (elachlan) [inline generation/WinSDK/enums.json:30383]: This should have a value of:
```
(DTT_TEXTCOLOR | DTT_BORDERCOLOR | DTT_SHADOWCOLOR | DTT_SHADOWTYPE | DTT_SHADOWOFFSET | DTT_BORDERSIZE | DTT_FONTPROP | DTT_COLORPROP | DTT_STATEID | DTT_CALCRECT | DTT_APPLYOVERLAY | DTT_GLOWSIZE | DTT_COMPOSITED)
```

**Review by sotteson1 — COMMENTED** [review]
The new system gets rid of the baseline .winmd, and instead updates a text file that will include the diff of your changes. (The output of the build is meant to help you automate your additions to this file.) Sorry to randomize your PR but I think this system will be a lot nicer for merges in the future.

**Review by elachlan — APPROVED** [review]

File: generation/WinSDK/enums.json [hunk 2]
```diff
@@ -30318,5 +30318,77 @@
...
+        "name": "DTT_CALLBACK",
+        "value": "0x1000"
+      },
+      {
+        "name": "DTT_COMPOSITED",
+        "value": "0x2000"
+      },
+      {
+        "name": "DTT_VALIDBITS",
+        "value": "(DTT_TEXTCOLOR | DTT_BORDERCOLOR | DTT_SHADOWCOLOR | DTT_SHADOWTYPE | DTT_SHADOWOFFSET | DTT_BORDERSIZE | DTT_FONTPROP | DTT_COLORPROP | DTT_STATEID | DTT_CALCRECT | DTT_APPLYOVERLAY | DTT_GLOWSIZE | DTT_COMPOSITED)"
```
Comment (riverar) [inline generation/WinSDK/enums.json:30384]: While this aligns with the header/docs, I'm curious about the missing `DTT_CALLBACK` bit in `DTT_VALIDBITS`. Is this a bug @mikebattista? (Not a blocker for this PR.)

**Review by riverar — APPROVED** [review]

File: generation/WinSDK/enums.json (hunk 2 above)
Comment (elachlan) [reply generation/WinSDK/enums.json:30384]: That's probably my fault. I copied it from the docs.
https://docs.microsoft.com/en-us/windows/win32/api/uxtheme/ns-uxtheme-dttopts

The code in the Uxtheme.h has this:
```C++
#define DTT_VALIDBITS       (DTT_TEXTCOLOR | \
                             DTT_BORDERCOLOR | \
                             DTT_SHADOWCOLOR | \
                             DTT_SHADOWTYPE | \
                             DTT_SHADOWOFFSET | \
                             DTT_BORDERSIZE | \
                             DTT_FONTPROP | \
                             DTT_COLORPROP | \
                             DTT_STATEID | \
                             DTT_CALCRECT | \
                             DTT_APPLYOVERLAY | \
                             DTT_GLOWSIZE | \
                             DTT_COMPOSITED)
```

So not sure if this is on purpose, the docs are wrong, or there is a bug in uxtheme?

Comment (riverar) [reply generation/WinSDK/enums.json:30384]: Yeah I suspect the header is bugged. Hoping Mike can file a bug internally.

**Pull request merged**